docs/sidebars.js

```diff
@@ -25,6 +25,15 @@ module.exports = {
       "behavior/lighting",
       "behavior/power",
     ],
+    Codes: [
```

Comment:
Just thought of this: Codes might not be super obvious at first that it contains all the defines for HID codes for keys. Maybe Keycodes would be more direct?

Reply:
I'm glad you spotted that!  I was hoping someone would bring it up because it was deliberate.  I started off with `Key Codes`, but as time went on and I thought through the bigger picture I realised that:
1. It might be a good section to include summary tables of the non-HID codes also (they can just be mdx pages).  Behavior codes are not currently the easiest to find buried inside the behavior pages, so it might be wise to turn each set of codes into its own mdx page which we can embedded under both sections (mdx pages can be used as components).  This was previously [discussed](https://discordapp.com/channels/719497620560543766/719565084208398406/765590462274469890).
2. We'll soon be moving beyond just key codes.  More generally it's `hid` codes and other behavior codes (which is partly why named the data file `hid.js` instead of `keycodes.js`).  The counterarguments to this weaken somewhat when you start considering encoders, trackballs, mice, sliders, touch controls, etc.
3. We're planning to refactor `keys.h` into `codes.h` in the [near future](https://discordapp.com/channels/719497620560543766/719565084208398406/769222960892346438) so that there's only one point of entry within user configs.
4. `codes` is sometimes used as a shorthand for `key codes` in discussions.